packages/react-ui/src/widgets/TableWidgetUI/TableWidgetUI.js

```diff
@@ -12,28 +12,28 @@ import {
 } from '@mui/material';
 
 import makeStyles from '@mui/styles/makeStyles';
+import styled from '@emotion/styled';
```

Comment:
import it from @mui/material

https://mui.com/system/styled/#import-path

Reply:
Yes, please 🙏🏻 and the makeStyles line 14 should be removed right?

Reply:
yes, indeed